client/src/components/Common/SideBar/index.jsx

```diff
@@ -116,9 +149,13 @@ const SIDEBAR_ITEMS = [
 	},
 ];
 
-const SideBar = ({ title, modules }) => {
+const SideBar = ({ modules }) => {
+	const url = window.location.pathname;
+	const title = url.split('/')[1];

```

Comment:
Should we handle the root URL (eg: casperdash.io)? We might return "dashboard" as default title.